src/components/__tests__/FontAwesomeIcon.test.js

```diff
@@ -3,18 +3,50 @@
  */
 
 import { library } from '@fortawesome/fontawesome-svg-core'
-import { faCoffee, faCircle, faAlien } from '../__fixtures__/icons'
-import { compileAndMount, mountFromProps, coreHasFeature, REFERENCE_ICON_USING_STRING, REFERENCE_ICON_BY_STYLE, ICON_ALIASES } from '../__fixtures__/helpers'
+import { faCoffee, faCircle, faAlien, faGlasses } from '../__fixtures__/icons'
+import { coreHasFeature, compileAndMount, mountFromProps, REFERENCE_ICON_BY_STYLE, ICON_ALIASES, REFERENCE_ICON_USING_STRING, REFERENCE_ICON_USING_FAMILY } from '../__fixtures__/helpers'
 import FontAwesomeIcon from '../FontAwesomeIcon'
 
 beforeEach(() => {
-  library.add(faCoffee, faCircle, faAlien)
+  library.add(faCoffee, faCircle, faAlien, faGlasses)
 })
 
 afterEach(() => {
   library.reset()
 })
 
+describe('using a family', () => {
+  if(coreHasFeature(REFERENCE_ICON_USING_FAMILY)) {
```

Comment:
Just a nit on spacing.

```suggestion
  if (coreHasFeature(REFERENCE_ICON_USING_FAMILY)) {
```